### PR TITLE
Do not fetch the inlined rule data multiple times

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

**Review by robnester-rh — APPROVED** [review]
LGTM

File: internal/policy/source/source_test.go [hunk 1]
```diff
@@ -103,19 +104,19 @@ func TestInlineDataSource(t *testing.T) {
...
+	require.FileExists(t, file)
 
 	data, err := os.ReadFile(file)
-	assert.NoError(t, err)
-	assert.Equal(t, []byte("some data"), data)
+	require.NoError(t, err)
+	require.Equal(t, []byte("some data"), data)
 
-	assert.Equal(t, "data:application/json;base64,c29tZSBkYXRh", s.PolicyUrl())
+	require.Equal(t, "data:application/json;base64,c29tZSBkYXRh", s.PolicyUrl())
```
Comment (simonbaird) [inline internal/policy/source/source_test.go:119]: What's the difference here?

Comment (simonbaird) [reply internal/policy/source/source_test.go:119]: I googled it: https://www.yellowduck.be/posts/assert-vs-require-in-testify

**Review by robnester-rh — COMMENTED** [review]
LGTM

File: internal/policy/source/source.go [hunk 2]
```diff
@@ -144,13 +144,21 @@ func InlineData(source []byte) PolicySource {
 }
 
 func (s inlineData) GetPolicy(ctx context.Context, workDir string, showMsg bool) (string, error) {
-	dest := uniqueDestination(workDir, s.Subdir(), fmt.Sprintf("%x", (sha256.Sum256(s.source))))
+	dest := path.Join(workDir, s.Subdir())
```
Comment (zregvart) [inline internal/policy/source/source.go:147]: With the use of `downloadCache` the previous `dest` using `uniqueDestination` can be used

Comment (lcarva) [reply internal/policy/source/source.go:147]: Just tried. It doesn't work because it makes the key unique on every invocation of `GetPolicy`.

**Pull request merged**